cluster-autoscaler/estimator/threshold.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package estimator
+
+import (
+	"time"
+
+	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
+)
+
+// Threshold provides resources configuration for threshold based estimation limiter.
+// Return value of 0 means that no limit is set.
+type Threshold interface {
+	GetNodeLimit(cloudprovider.NodeGroup, *EstimationContext) int
+	GetDurationLimit() time.Duration
```

Comment:
Why `GetDurationLimit` does not get the same `cloudprovider.NodeGroup, *EstimationContext` arguments? Currently there is no implementation that needs them, but it seems reasonable to include them as well

Reply:
Amended in f3dfeeeb47a021ca4b854be5363e1a1663313b3b